### DISCUSSION TITLE
What do you use jrnl for?

### DISCUSSION THREAD

**Discussion by micahellison** [jrnl-org/jrnl#1306] [General]
We don't track any user activity, which I think is a great feature about jrnl, but the result of that is that we don't know what people are using jrnl for these days.

I'd love to hear examples of what you're doing with it. Do you have templates you like to use? Do you sync and backup your journal, and if so, how? What sort of devices are you using it on? Are you using a variety of different journals for different topics? If so, what topics? Do you use it professionally, for self-improvement, or do you simply journal out of compulsion?

If you'd like to let us know but don't want to post about it publicly, feel free to email us directly at: [jrnl-sh@googlegroups.com](mailto:jrnl-sh@googlegroups.com).

**Comment by micahellison** [comment]
As for me, one of my favorite uses of jrnl is for home cooking. I keep a cooking journal much like a lab notebook: I type up or copy-paste what I plan to do, record the measurements and procedures I end up using (as they often differ from the plan in the heat of the moment), then I add to the entry after eating to record how I felt about the meal, along with ideas for how to improve it in the future.

Then, the next time I cook the same sort of meal, I review the last few entries about it with a simple text search, like so:
```
jrnl cooking -contains chili
```
I've been thinking about someday keeping a more static set of plaintext recipe files, then using the new-ish `--config-override` argument to change the template to the recipe that I'm cooking when I start a new entry, but so far I like constantly iterating my most recent recipes rather than having definitive versions.

**Comment by giuliofoletto** [comment]
I've just started using jrnl, so this might change, but I currently use it as a log of everything that I do or happens in my life, I think of it as the Captain's log from Star Trek. I know this is trivial and not particularly imaginative, but I really needed a tool to record thoughts and events without worrying about complicated formatting, or file structures, or relations between entries... With jrnl I can just write down stuff as soon as it happens with minimal friction. Thank you very much for this great tool.
